README.md

```diff
@@ -1,3 +1,438 @@
-Partners are maintained as individual branches.
+# Qualcomm FoundriesFactory Reference Manual
 
-Choose a branch to view the LmP partner integration content.
+## Table of Contents
+
+- Document Revision History
```

Comment:
I think we can add links to the sections below like:
```
[Document Revision History](#document-revision-history)
```

But this needs to be tested

Reply:
you can view the rendered version here https://github.com/foundriesio/meta-partner/blob/c4168e5d0a3c1aeabb98f5f9250afb1c18700a52/README.md there's a couple of odd formatting things.